pythonosc/osc_message_builder.py

```diff
@@ -72,6 +75,8 @@ def add_arg(self, arg_value, arg_type=None):
         arg_type = self.ARG_TYPE_TRUE
       elif arg_value == False:
         arg_type = self.ARG_TYPE_FALSE
+      elif arg_value is None:
```

Comment:
I think despite the confusing name, `Infinitum` is actually meant to be an "Impulse" type without any data, for sending a trigger signal (see [here](https://community.etcconnect.com/control_consoles/f/eos-family/18865/what-is-osc#70351)).